server/routerlicious/packages/services-shared/src/livenessMonitor/livenessMonitor.ts

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ILivenessMonitor } from "@fluidframework/server-services-core";
+import { LumberEventName, Lumberjack } from "@fluidframework/server-services-telemetry";
+import { Router } from "express";
+import { handleResponse } from "../http";
+
+export class LivenessMonitor implements ILivenessMonitor {
+	public constructor(
+		private readonly checkList: (() => Promise<void>)[],
```

Comment:
The key is to inject all different checks by each individual service to here. Using this PR for example, alfred added redis ping check and oldest op check